src/CommonITILActor.php

```diff
@@ -152,6 +152,7 @@ public function post_deleteFromDB()
                 && ($item->countGroups(self::ASSIGN) === 0)
                 && ((int) $item->fields['status'] !== CommonITILObject::CLOSED)
                 && ((int) $item->fields['status'] !== CommonITILObject::SOLVED)
+                && !isset($this->input['_do_not_compute_status'])
```

Comment:
I am not sure this is related to this PR.

Reply:
This prevents the ticket status from changing temporarily to new during the short period when there are no technicians, between deletion and addition.